app/content/css/styles.css

```diff
@@ -276,11 +277,10 @@
   font-size: 22px;
 }
 
-
 #tables_wrapper {
   border: 1px solid #b4b4b4;
   overflow: hidden;
-  height: calc(100% - 42px);
+  /*height: calc(100% - 42px);*/
```

Comment:
having a comment why this commented line is still needed is generally a good thing, otherwise you could remove it completely. Git history allow to restore any changes :)